docs/app/_components/Navigation.tsx

```diff
@@ -47,17 +47,19 @@ export const useNavigation = (): NavigationSection[] => {
       subsections: subsections.map(
         (subsection: { name: string; slug: string }) => ({
           name: subsection.name,
-          links: allContentPages
-            .filter(
-              // Subsection Page = has a section AND a subsection
-              page =>
-                page.section === slug && page.subsection === subsection.slug
-            )
-            .map(page => ({
-              name: page.title,
-              href: `/${page.slug}`,
-              badge: page.badge,
-            })),
+          links: [
+            ...allContentPages
```

Comment:
why do we need to destructure here? :-o

Reply:
oh we don't need it was from when I had two page types